requirements.txt

```diff
@@ -38,3 +38,4 @@ wcwidth==0.2.4
 Werkzeug==1.0.1
 wrapt==1.12.1
 zipp==3.1.0
+psycopg2-binary==2.8.5
```

Comment:
I don't believe you need this. Unless this is a Mac thing? (I forget which system you're on.)

Reply:
Apparently, for mac I do need this because without it it wouldn't be able to find the app.py

Reply:
Alright. I'll test later to make sure it doesn't break anything on windows. Likely won't. If it does we can split the requirements.txt for windows and mac.